etna/models/base.py

```diff
@@ -146,3 +148,82 @@ def _build_models(self):
         self._models = {}
         for segment in self._segments:
             self._models[segment] = deepcopy(self._base_model)
+
+
+class FitAbstractModel(ABC):
+    """Interface for model with fit method."""
+
+    @abstractmethod
+    def fit(self, ts: TSDataset) -> "FitAbstractModel":
+        """Fit model.
+
+        Parameters
+        ----------
+        ts:
+            Dataset with features
+
+        Returns
+        -------
+        result:
```

Comment:
I am not sure about the word "result", may be it is better to replace it with "self"